CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Added
+
+* BackupPolicy to manage backups supporting scheduling and retention configuartion. 
+* BackupPolicy Readme explaining the feature
+* BackupPolicy API Documentation
+* end-to-end on BackupPolicy RetentionPolicy
```

Comment:
I think this section would be unclear to a user.

I would expect an explanation of what the new feature does. For example: 
> BackupPolicy feature: Create new backups based on a schedule. You can now configure a schedule for each Postgresql instance's backups and when the backups will be cleaned up.

Please remove the mentions of end-to-end tests. Users are not concerned about that.